package.json

```diff
@@ -7,7 +7,7 @@
     "example": "example"
   },
   "engines": {
-    "node": "8 || 9 || 10 || 11 || 12 || 13 || 14 || 15 || 16 || 17 || 18 || 19"
+    "node": "8 || 9 || 10 || 11 || 12 || 13 || 14 || 15 || 16 || 17 || 18 || 19 || 20"
```

Comment:
@achrinza any reason why you can't put `>=14` here so it will work for **21**, **22**, ... too?